doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -116,72 +110,107 @@ The presence of a ``bicr`` node in the application devicetree will automatically
+      curl -LO https://files.nordicsemi.com/artifactory/swtools/external/scripts/nrf54h20es_trim_adjust.jlink
 
-Switch LCS to RoT
-=================
+#. Run the script::
+
+      JLinkExe -CommanderScript nrf54h20es_trim_adjust.jlink
+
+Verify the LCS and transition to RoT
+************************************
+
+To successfully run your custom application on your custom board, the SoC must have its Lifecycle State (LCS) set to ``RoT`` (meaning Root of Trust).
+If the LCS is set to ``EMPTY``, you must transition it to ``RoT``.
+
+.. note::
+   The forward transition to LCS ``RoT`` is permanent.
+   After the transition, it is not possible to transition backward to LCS ``EMPTY``.
 
 To transition the LCS to ``RoT``, do the following:
 
-1. Program BICR and secdom::
+1. Verify the current lifecycle state of the nRF54H20::
+
+      nrfutil device x-adac-discovery --serial-number <serial_number>
+
+   The output will look similar to the following::
+
+      *serial_number*
+      adac_auth_version     1.0
+      vendor_id             Nordic VLSI ASA
+      soc_class             0x00005420
+      soc_id                [e6, 6f, 21, b6, dc, be, 11, ee, e5, 03, 6f, fe, 4d, 7b, 2e, 07]
+      hw_permissions_fixed  [00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+      hw_permissions_mask   [01, 00, 00, 00, 87, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+      psa_lifecycle         LIFECYCLE_EMPTY (0x1000)
+      sda_id                0x01
+      secrom_revision       0xad3b3cd0
+      sysrom_revision       0xebc8f190
+      token_formats         [TokenAdac]
+      cert_formats          [CertAdac]
+      cryptosystems         [Ed25519Sha512]
+      Additional TLVs:
+      TargetIdentity: [ff, ff, ff, ff, ff, ff, ff, ff]
 
-      nrfjprog -f nrf54h --coprocessor CP_SECURE --program /*insert_path_to_your_build_directory_here*/secdom/src/secdom-build/zephyr/zephyr.hex --verify
-      nrfjprog -f nrf54h --coprocessor CP_SECURE --program bicr.hex --verify
+#. If the lifecycle state (``psa_lifecycle``) shown is ``RoT`` (``LIFECYCLE_ROT (0x2000)``), no LCS transition is required.
+   If the lifecycle state (``psa_lifecycle``) shown is not ``RoT`` (``LIFECYCLE_EMPTY (0x1000)`` means the LCS is set to ``EMPTY``), set it to Root of Trust using the following command::
 
-#. Run these commands::
+      nrfutil device x-adac-lcs-change --life-cycle rot --serial-number <serial_number>
 
-      nrfjprog --coprocessor CP_SECURE --memwr 0x0E000108 --val 0x40000
-      nrfjprog --coprocessor CP_SECURE --memwr 0x0E00010C --val 0x0E003000
+#. Verify again the current lifecycle state of the nRF54H20::
 
-      nrfjprog --family nrf54h --bootmode ROM
-      nrfjprog --family nrf54h --adac lcs_change PSA_ROT_PROVISIONING --single-step
-      nrfjprog --family nrf54h --bootmode NORMAL
+      nrfutil device x-adac-discovery --serial-number <serial_number>
 
-#. Flash your application using west::
+   The output will look similar to the following::
```

Comment:
We mostly use `.. code-blocks::` instead of `::`, so I would suggest aligning to that.